package.json

```diff
@@ -23,6 +23,7 @@
     "@tiptap/extension-link": "^2.0.0-beta.220",
     "@tiptap/react": "^2.0.0-beta.220",
     "@tiptap/starter-kit": "^2.0.0-beta.220",
+    "axios": "^1.3.4",
```

Comment:
If your tests are not passing locally try adding this to your package.json:
```'
  "jest": {
    "moduleNameMapper": {
      "axios": "axios/dist/node/axios.cjs"
    }
```
[here the link to the source](https://stackoverflow.com/questions/73958968/cannot-use-import-statement-outside-a-module-with-axios)